pyworkflow/template.py

```diff
@@ -108,13 +111,36 @@ def _replaceFields(self):
         for field in self.params.values():
             self.content[field.getIndex()] = field.getValue()
 
+    def getParams(self):
+        return self.params
+
+    def setParamValue(self, alias, newValue):
+        paramsSetted = 0
+        for field in self.params.values():
+            if field.getAlias() == alias:
+                oldValue = field.getValue()
+                field.setValue(newValue)
+                if field.validate():
+                    paramsSetted += 1
+                    print(greenStr("Setted the parameter (%s). New value: %s") %
```

Comment:
Gramatically incorrect. I would change it by something like "Parameter %s was set to %s"